README.md

```diff
@@ -66,12 +93,116 @@ You'll also need to create some (human) users.  The email address of the new use
 should match the Google account address used to log in.  See below for instructions
 on how to do this via the commandline.
 
-### OSMOD commands
+#### Run in a docker container
+
+To run the service in a local docker container, run the following commands:
+
+```bash
+# Make sure any local instances of MySQL and Redis are not running
+# E.g., on Ubuntu, stop the services
+sudo systemctl stop mysql.service redis_6379.service redis-server.service
+
+# Create docker images and launch the service
+docker-compose -f deployments/local/docker-compose.yml up -d
+
+# Initialise the database
+mysql -u $DATABASE_USER -p -h 127.0.0.1 $DATABASE_NAME < packages/backend-core/seed/initial-database.sql
+./bin/osmod migrate
+```
+
+To shut down the service and delete all your containers:
+
+```bash
+docker-compose -f deployments/local/docker-compose.yml down
+```
+
+And to see what the container is doing:
+
+```bash
+docker-compose -f deployments/local/docker-compose.yml logs
+```
+
+#### Adding users
+
+To actually do anyting, you'll need to create some users.  You'll need at least
+
+* a human user so you can log in and view comments
+
+* a service user that knows how to talk to a PerpectiveAPI proxy server.
+
+Add human users with the following command:
+
+```bash
+./bin/osmod users:create --group general --name "Name" --email "email@gmail.com"
+```
+
+Human users get authenticated using Google's OAuth server, so the email address must
+correspond to a google account.
+
+Add the proxy service user with the following command:
 
 ```bash
-./bin/osmod ...
+./bin/osmod users:create --group service --name "PerspectiveProxy" --endpoint=<proxy URL>
 ```
 
+where `<proxy URL>` is the URL of [The Perspective API proxy] you plan on using.
+
+### The Perspective API proxy
+
+OSMod is designed to talk to the Perspective API via a proxy server: the
```

Comment:
Maybe note also that we plan to remove this and package the proxy with Omod?

Reply:
Actually, I decided not to eliminate the proxy entirely, as that is what NYT is currently using, and I didn't want to mess them around too much.  It is now configurable.  I'll queue up that change for review next.

That change rewrites this text fragment a little bit.  So we can update it as you see fit in that review.